README.md

```diff
@@ -1,38 +1,44 @@
 
-## Join the Globalping Probe Community - [Learn more about it](https://github.com/jsdelivr/globalping)
+### Set up a virtual probe
+To join the Globalping probe network, all you need to do is run our Docker container, which works on both x86 and ARM architectures. 
+
+Use the following command:
 
-To join the Globalping probe network all you have to do is run our container.
 
 ```
 docker run -d --log-driver local --network host --restart=always --name globalping-probe ghcr.io/jsdelivr/globalping-probe
 ```
-The container will work on both x86 and ARM architectures.
 
-Once you connect you will become part of the global community that powers the [Globalping Platform](https://github.com/jsdelivr/globalping)
+> [!TIP]
+> For automation, consider using our [universal installation script for Linux servers](https://gist.github.com/jimaek/7b8312c2c37f9002a5cc0108ebfd43e1) and adapt it as needed.
 
----
-#### Podman alternative
-Note that you also need to [install a service](https://linuxhandbook.com/autostart-podman-containers/) to make sure the container starts on boot. Additionally check the logs of the container to ensure it started successfully, podman can sometimes break things by requiring additional permissions.
+### Podman alternative
+For users opting for Podman, [follow the instructions here](https://linuxhandbook.com/autostart-podman-containers/) to make sure the container automatically starts on boot. Also, check the container logs to see if it started successfully, as Podman may still require additional permissions.
+Run the container with the following command:
 ```
 sudo podman run --cap-add=NET_RAW -d --network host --restart=always --name globalping-probe ghcr.io/jsdelivr/globalping-probe
 ```
----
-For automation purposes consider using this template of a [universal installation script for Linux servers](https://gist.github.com/jimaek/7b8312c2c37f9002a5cc0108ebfd43e1).
 
 ## Where to run
+You can run the virtual probe on anything that can run a Docker container, supporting x86 and ARM architectures. This includes any Linux server hosted with a cloud provider, your home server, or even a Raspberry Pi lying around at your office. 
 
-You can run it on anything that can run a docker container. Any kind of linux server hosted with a cloud provider, your home server or even a Raspberry Pi that you have lying around. There is nothing to configure, simply run the container.
-The only requirement is a stable internet connection.
+No configuration is needed; just run the container with a stable internet connection.
 
-## Updating
+## How to update 
+You don't need to worry about updates as the probe automatically updates to the latest version, pulled directly from GitHub.
 
-You don't need to worry about updates. Our probe will automatically update to the latest version as they become available. It will be pulled directly from GitHub and installed within the container. The container itself won't get updated, only the code it's running. 
```

Comment:
I think the previous sentence is better because it explains how exactly it updates, which some technical users will appreciate

Reply:
Updated this section to be more similar to the original. I also added a "Note" alert with info about code and container updates